courts_db/data/courts.json

```diff
@@ -6289,7 +6289,7 @@
         "examples": [],
         "court_url": "http://www.neb.uscourts.gov/",
         "type": "bankruptcy",
-        "id": "nebraskab",
```

Comment:
Unfortunately, your change in Nebraska makes sense, but we already have nebraskab - in our CL system.  I don't think the plan would be to change the court codes already in the system.

Reply:
Makes sense.  I'm still relatively green with this process but should I resubmit this pull request with only the changes for Hawaii and Texas?

Reply:
The easiest thing is to make the change, add another commit, and push it to this branch. That'll update this PR.